### ISSUE TITLE
lammps file.data to gromacs files error: intermol.exceptions.UnimplementedSetting: ['pair_style', 'none'] has not yet been implemented in InterMol for LAMMPS.

### ISSUE THREAD

**Issue by ananvodo** [shirtsgroup/InterMol#345]
Hello 

I am running convert.py using the following command:

`python intermol/convert.py --lmp_in /to_directory/mont0_333_K_15_single_layer.data --gromacs --odir test_output`

I have tried with python 2.7 and 3.6 and all of them give me the same error:

`intermol.exceptions.UnimplementedSetting: ['pair_style', 'none'] has not yet been implemented in InterMol for LAMMPS.`

Can anyone help me if this is an error or it just has not been implemented? 
When I run the test_all.py I do not have any problem (that means my intermol is well installed).

The link to my lammp_file.data is this:
[](https://drive.google.com/open?id=1M26n4fhijVpiqvbeRLTKCNeqxvBcXwGe)

Thanks

**Comment by ctk3b** [comment]
Yup that error means exactly that: we haven't implemented parsing of data files if `pair_style` isn't defined.

The usual way that we load LAMMPS files is via an input file that itself loads the data file. 

See this example: https://github.com/shirtsgroup/InterMol/blob/master/intermol/tests/lammps/unit_tests/angle_permute-1_vacuum/angle_permute-1-data_vacuum.input#L14

**Comment by ananvodo** [comment]
Thanks for your response.

As a question, do you have some idea when it might be implemented?

**Comment by mrshirts** [comment]
Not clear.  There is currently no financial support for InterMol, so we have no one on it.  MolSSI may be developing a replacement, but that is currently slow.

**Comment by H-Hariman** [comment]
Hello,
I have similar issue with the attached single-simple molecule to convert lammps to gromacs;
I would be so appreciated if you could give a try to see if the same issue would appear from your side.

[MOL.input.txt](https://github.com/shirtsgroup/InterMol/files/4458275/MOL.input.txt)
[MOL.lmp.txt](https://github.com/shirtsgroup/InterMol/files/4458276/MOL.lmp.txt)
[mol.log.txt](https://github.com/shirtsgroup/InterMol/files/4458277/mol.log.txt)

Thanks

**Comment by mrshirts** [comment]
We are unlikely to implement new functionality in the next few months.

**Comment by yel21** [comment]
Dear Intermol users, 

Have anyone had an opportunity to solve this problem yet? 

Thanks
